dht.go

```diff
@@ -2,9 +2,8 @@ package dht
 
 import (
 	"fmt"
+	"periph.io/x/host/v3"
```

Comment:
Please keep external imports at the end.

Reply:
This does not comply with the Golang coding standard formatting.

Reply:
I believe if you leave the new line between, the Go formatting will leave the newline there.

Reply:
I didn't know that existed in Go. I applied your suggested changes.